guides/common/modules/con_performing-post-upgrade-tasks.adoc

```diff
@@ -4,3 +4,7 @@
 Some of the procedures in this section are optional.
 You can choose to perform only those procedures that are relevant to your installation.
 
+ifdef::Satellite[]
+If you use custom templates created from a cloned default template, create the custom template again after the upgrade.
```

Comment:
Is re-creating custom templates a requirement? If so, perhaps state it like this: "If you use a custom template cloned from a default template, you must re-create the custom template after the upgrade."